<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationLogDispKSK</class>
 <widget class="QWidget" name="StationLogDispKSK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StationLogDispKSK</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(220, 220, 220);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>64</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="adjPreviewBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>预告(F1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="agreeAdjBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>同意(F2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="arrivalBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>到达(F3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="departureBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>出发(F4)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="throughBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>通过(F11)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="adjAgrBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>邻站(F5)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelPickUPBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>取消接车
(Esc)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBlockBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>取消闭塞(F6)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelDepartBtn">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>56</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>取消发车(F7)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="tableHeadFrame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>24</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>24</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>中间站行车日志[]</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>8</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>148</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>148</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>2024年 09月09日 星期一</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>天气</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>24</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>晴</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>阴</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>多云</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>大风</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>小雨</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>大雨</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>小雪</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>大雪</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>冰雹</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>雾天</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <property name="spacing">
                 <number>2</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text">
                   <string>值班时间</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_5">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_6">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="label_7">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text">
                   <string>值班人</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="label_9">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_10">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QLabel" name="label_11">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="tableFrame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CTCWindows::TrafficMsgLog" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>220</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>220</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CTCWindows::TrafficMsgLog</class>
   <extends>QWidget</extends>
   <header location="global">CTCMainWindow/CommonWidget/trafficmsglog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
